CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2021-10-06 Plugin Parameters
+<p>If you enter a value into a number parameter field that is either less than the minimum allowed or more than maximum allowed, the field turns red.</p>
```

Comment:
I figured most probably don't have the logs open all the time so omitted a mention of that somewhat intentionally. I would be cautious to say "and a notification is displayed" anyway though as it may suggest a popup or something else being displayed to the user rather than merely a log entry